docusaurus/sidebars-ios.json

```diff
@@ -22,6 +22,15 @@
         "controllers/messages"
       ]
     },
+    {
+      "Controllers with Combine" : [
+        "combine/combine-overview",
+        "combine/connection",
+        "combine/channels",
+        "combine/users",
+        "combine/messages"
```

Comment:
we need one for events as well

Reply:
(we allow you to subscribe to events with combine publishers)

Reply:
Thanks for this input, added one for events as well!